client/scss/components/_userbar.scss

```
@@ -261,6 +261,103 @@ $positions: (
   }
 }
 
+.w-dialog--userbar {
+  // Display off to the side of the page rather than in the middle.
+  inset-inline-start: auto;
+
+  .w-dialog__close-button {
+    $size: theme('spacing.6');
+    width: $size;
+    height: $size;
+    top: calc(-1 * $size / 2);
+    inset-inline-end: calc(-1 * $size / 2);
+    border-radius: theme('borderRadius.full');
+    border: 2px solid theme('colors.primary.DEFAULT');
```

```suggestion
    border: 2px solid theme('colors.primary.DEFAULT');
    background-color: theme('colors.white.DEFAULT');
```

And additionally, we need to change the styles on dialog’s `&__box` to not have a `overflow: hidden;`